misc.php

```diff
@@ -7,6 +7,11 @@
 License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
 */
 
+add_filter( 'upload_mimes', function( $mimes ) {
+	unset( $mimes['flv'] );
+	return $mimes;
+});
```

Comment:
Should we set the priority really high, so that other client code can't come along and re-enable it?